array-manipulation-1/src/initial.ts

```diff
@@ -0,0 +1,10 @@
+/* exported initial */
+function initial(array: unknown[]): unknown[] {
+  const newArray = [];
+  if (array.length > 1) {
+    for (let i = 0; i < array.length - 1; i++) {
```

Comment:
This line 4 `if (array.length > 1)` isn't necessary because you already have your `condition statement` in your `for` loop! If you array has a length of 0, it will evaluate is `0 < -1` which is false and then we'll skip past loop's code block.